DuckDuckGo.Fluent.Plugin/DuckDuckGo.Fluent.Plugin.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0-windows10.0.19041</TargetFramework>
+        <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
```

Comment:
Why?

Reply:
Visual Studio was giving some random errors earlier, will remove it now.